harry-core/pom.xml

```diff
@@ -58,18 +58,6 @@
             <artifactId>jackson-annotations</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>org.apache.cassandra</groupId>
-            <artifactId>cassandra-dtest-shaded</artifactId>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.apache.cassandra</groupId>
-            <artifactId>dtest-api</artifactId>
-            <scope>test</scope>
-        </dependency>
-
```

Comment:
this allows for building the project without pre-building the dtest shaded library, for execution against an external cluster